solr/solr-ref-guide/modules/deployment-guide/pages/solr-in-docker.adoc

```diff
@@ -342,6 +342,13 @@ Usage: jattach <pid> <cmd> [args ...]
     jcmd : execute jcmd command
 ....
 
+To find out the PID of Solr you can execute the following command and pick the `java` entry from the results:
```

Comment:
Can it be assumed that `grep` is available? If it was available (I don't know if it is or isn't) then could also use `ps -ef | grep java` to reduce the list.

Reply:
you know, I almost suggested that!

Reply:
Sure, using `grep` has only one result. I updated the PR.